lib/std/fs/Dir.zig

```diff
@@ -548,7 +602,16 @@ pub const Iterator = switch (builtin.os.tag) {
                     .SYMBOLIC_LINK => .sym_link,
                     .REGULAR_FILE => .file,
                     .SOCKET_STREAM, .SOCKET_DGRAM => .unix_domain_socket,
-                    else => .unknown,
+                    else => query_kind: {
```

Comment:
The wasi `filetype_t` enum has `UNKNOWN` as well, so this could also use that.

Reply:
oh, missed that. thx